plugin/sdk_v2/shared/runtime/system_info.py

```
@@ -0,0 +1,55 @@
+        timeout_ok = self._validate_timeout(timeout)
+        if isinstance(timeout_ok, Err):
+            return cast(Result[JsonObject, SystemInfoErrorLike], timeout_ok)
+        try:
+            return cast(Result[JsonObject, SystemInfoErrorLike], await self._impl.get_server_settings(timeout=timeout))
+        except Exception as error:
+            return cast(Result[JsonObject, SystemInfoErrorLike], Err(error if isinstance(error, SdkError) else TransportError(str(error), op_name="system_info.get_server_settings", timeout=timeout)))
+
+    async def get_python_env(self) -> Result[JsonObject, TransportError]:
+        try:
+            return await self._impl.get_python_env()
+        except Exception as error:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)